mappings/net/minecraft/util/math/BlockPos.mapping

```diff
@@ -98,6 +98,7 @@ CLASS net/minecraft/class_2338 net/minecraft/util/math/BlockPos
 		ARG 1 pos2
 	METHOD method_23627 stream (Lnet/minecraft/class_3341;)Ljava/util/stream/Stream;
 		ARG 0 box
+	METHOD method_25503 toMutable ()Lnet/minecraft/class_2338$class_2339;
```

Comment:
Not really, this is `mutableCopy` as the `Mutable` subclass does not have an implementation that returns itself.